apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/(chainPage)/components/client/FaucetButton.tsx

```diff
@@ -13,6 +13,7 @@ import { toUnits } from "thirdweb";
 import type { ChainMetadata } from "thirdweb/chains";
 import { useActiveAccount, useWalletBalance } from "thirdweb/react";
 import { mapV4ChainToV5Chain } from "../../../../../../../contexts/map-chains";
+import type { CanClaimResponseType } from "../../../../../../api/testnet-faucet/can-claim/route";
```

Comment:
importing from api route feels dirty, move the type to a shared types.ts file?

Reply:
moved to new page